tests/testVisitInfo.py

```diff
@@ -33,10 +33,10 @@
 import lsst.afw.image as afwImage
 
 RotTypeEnumNameDict = {
-    afwImage.RotType_UNKNOWN: "UNKNOWN",
-    afwImage.RotType_SKY: "SKY",
-    afwImage.RotType_HORIZON: "HORIZON",
-    afwImage.RotType_MOUNT: "MOUNT",
+    afwImage.RotType.UNKNOWN: "UNKNOWN",
+    afwImage.RotType.SKY: "SKY",
+    afwImage.RotType.HORIZON: "HORIZON",
+    afwImage.RotType.MOUNT: "MOUNT",
```

Comment:
I'm firmly in favor of the change. I don't like having two ways to do something and the SWIG way was ugly. I don't think it will be hard to propagate the change. But I understand it adds work and may need to be discussed more widely.